h5io/_h5io.py

```diff
@@ -193,14 +188,20 @@ def _triage_write(key, value, root, comp_kw, where,
                       where + '.csr_matrix_shape', cleanup_data=cleanup_data,
                       slash=slash)
     else:
-        if isinstance(value, (DataFrame, Series)):
-            if isinstance(value, DataFrame):
-                title = 'pd_dataframe'
-            else:
-                title = 'pd_series'
-            rootname = root.name
-            cleanup_data.append({title: (rootname, key, value)})
+        try:
+            from pandas import DataFrame, Series
+        except ImportError:
+            pass
         else:
+            if isinstance(value, (DataFrame, Series)):
+                if isinstance(value, DataFrame):
+                    title = 'pd_dataframe'
+                else:
+                    title = 'pd_series'
+                rootname = root.name
+                cleanup_data.append({title: (rootname, key, value)})
+                return
+
             err_str = 'unsupported type %s (in %s)' % (type(value), where)
             raise TypeError(err_str)
```

Comment:
These two lines need to be un-indented now two levels fro this to function properly